### PR TITLE
Add redis tester

### PR DESCRIPTION
**Pull request by ankrgyl**
A simple script that connects to Redis in a similar manner to our API server, which you can use to debug connections.

### CODE REVIEW

File: connection-testers/redis.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    });
+  } else {
+    throw new Error("Must either set REDIS_URL or REDIS_HOST");
+  }
+
+  await redisClient.connect();
+  await redisClient.setEx("connection-tester-test", 60, "test");
+  const value = await redisClient.get("connection-tester-test");
+  console.log("Connection test successful! (test value: ", value, ")");
+}
```
Comment (mdeeks) [inline connection-testers/redis.ts:28]: Needs a quit or it hangs
```
await redisClient.quit();
```

Comment (mdeeks) [reply connection-testers/redis.ts:29]: Nvm I pushed it

**Review by mdeeks — APPROVED** [review]
LGTM

**Comment by ankrgyl** [comment]
Oops, thank you!

**Pull request merged**